src/common/expression/AttributeExpression.cpp

```diff
@@ -27,13 +28,13 @@ const Value &AttributeExpression::eval(ExpressionContext &ctx) {
         result_ = lvalue.getVertex().vid;
         return result_;
       }
-      for (auto &tag : lvalue.getVertex().tags) {
-        auto iter = tag.props.find(rvalue.getStr());
-        if (iter != tag.props.end()) {
-          return iter->second;
-        }
+      const auto &tags = lvalue.getVertex().tags;
+      auto iter = std::find_if(tags.begin(), tags.end(), la);
+      if (iter == tags.end()) {
+        return Value::kNullValue;
       }
-      return Value::kNullValue;
+      result_.setMap(Map(iter->props));
```

Comment:
Add some tests for returning the tag of a vertex, such as:

```
match (v:player{name: 'Tim Duncan'}) return v.player
```

Reply:
Ok.